packages/lib/package.json

```diff
@@ -29,12 +29,12 @@
     "clean": "rimraf 'build/*' 'coverage/*'",
     "build": "run-s 'build:*'",
     "build:js": "run-s 'build:js:*'",
-    "build:js:bundle": "rollup -c",
+    "build:js:bundle": "webpack --mode production --config-name lib",
     "build:js:minify": "uglifyjs build/dist/Miew.js -o build/dist/Miew.min.js -mc drop_console=true --comments /copyright/i --source-map \"content='build/dist/Miew.js.map',url='Miew.min.js.map'\"",
```

Comment:
Ideally, we don't need a separate minification stage if we use webpack. Everything can be done with the webpack and its plugins.